src/pipelines/epidemiology/at_2_authority.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict
+from pandas import DataFrame
+from lib.data_source import DataSource
+from lib.time import datetime_isoformat
+from lib.utils import table_merge
+
+
+class AustriaLevel2DataSource(DataSource):
+    def parse_dataframes(
+        self, dataframes: Dict[str, DataFrame], aux: Dict[str, DataFrame], **parse_opts
+    ) -> DataFrame:
+
+        data = table_merge(
+            [
+                dataframes["confirmed_deceased_recovered"].rename(
+                    columns={
+                        "AnzahlFaelle": "new_confirmed",
+                        "AnzahlFaelleSum": "total_confirmed",
+                        "AnzahlTotTaeglich": "new_deceased",
+                        "AnzahlTotSum": "total_deceased",
+                        "AnzahlGeheiltTaeglich": "new_recovered",
+                        "AnzahlGeheiltSum": "total_recovered",
+                    }
+                ),
+                dataframes["tested"].rename(
+                    columns={
+                        "TestGesamt": "total_tested",
+                        "MeldeDatum": "Time"
+                    }
+                )
+            ],
+            how="outer",
+        )
+
+        # Convert date to ISO format
+        data["date"] = data["Time"].apply(
+            lambda x: datetime_isoformat(x, "%d.%m.%Y %H:%M:%S"))
+
+        # Create the key from the state ID
+        data["key"] = data["BundeslandID"].apply(lambda x: f"AT_{x}")
+
+        # Drop whole-country data
+        data = data[data["key"] != "AT_10"]
```

Comment:
Why are you dropping whole country data? Typically, data from the health authority is the preferred one and it also tends to be more timely.

Reply:
I didn't see whole-country data for other countries and assumed that we just added up the regions to get it.  I could report it if that's better (should we also be adding up the individual states of other countries as well to report their whole-country data?)

Reply:
Yes, we should try to add individual states wherever possible. Generally if there's a value for whole country data (like here) that's *highly* preferred since it tends to be much more accurate.

Reply:
Added whole country data.  knowledge_graph.csv already has both the NUTS/2 and ISO-3166-2 keys, with the NUTS/2 rows having the datacommons field populated but the ISO-3166-2 rows not having the datacommons field populated.  Does anything need to change?

Reply:
Got it. In that case, adding the datacommons ID using `ISO/<code>` (like `ISO/AT-1`) would be ideal.

Reply:
Updated -- let me know if that's what you meant